examples/pytorch/ogc/README.md

```diff
@@ -0,0 +1,37 @@
+# Optimized Graph Convolution (OGC)
+
+This DGL example implements the OGC method from the paper: [From Cluster Assumption to Graph Convolution: Graph-based Semi-Supervised Learning Revisited](https://arxiv.org/abs/2309.13599).
+With only one trainable layer, OGC is a very simple but powerful graph convolution method.
+
+
+## Example Implementor
```

Comment:
Can you mention in Readme about what dgl version has been tested with this example?

Reply:
Thanks. We use DGL 1.1.2, and we have clearified all the dependencies in the readme file.